sonic-slave-buster/Dockerfile.j2

```diff
@@ -398,15 +484,19 @@ RUN sudo augtool --autosave "set /files/etc/dpkg/dpkg.cfg/force-confdef"
 ## do not modify the current configuration file, the new version is installed with a .dpkg-dist suffix
 RUN sudo augtool --autosave "set /files/etc/dpkg/dpkg.cfg/force-confold"
 
+{%- if CROSS_BUILD_ENVIRON != "y" %}
 # For linux build
 RUN apt-get -y build-dep linux
+{%- else %}
+RUN apt-get source kernel-wedge && sudo apt-get build-dep -a$arch kernel-wedge && cd kernel-wedge-2.99/ && CONFIG_SITE=/etc/dpkg-cross/cross-config.$arch  DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -a$arch -Pcross,nocheck && cd .. && sudo dpkg -i kernel-wedge_2.99_all.deb
```

Comment:
`kernel-wedge` appears to be an arch-independent package. Why does it need to be packaged again here?